Makefile

```diff
@@ -41,6 +41,13 @@ upgrade-test: DOCKER_IMAGE?=$(shell ./scripts/fallback-docker.sh $(LOCAL_DOCKER_
 upgrade-test:
 	go test -v ./pkg/tests/upgrade_tests/ -timescale-docker-image=$(DOCKER_IMAGE)
 
+.PHONY: jaeger-storage-test
+jaeger-storage-test: CURRENT_BRANCH?=$(shell git branch --show-current | sed 's#/#-#')
```

Comment:
It would be nice to reuse target variables since they are repeated among different targets. Couldn't we just define them on top?